test/private/execute_hash_stage.m

```diff
@@ -28,7 +28,7 @@
     % By storing the hash table in a persistent variable, the amount of disk
     % operations is minimized (i.e. reading the file, parsing it and storing its
     % data in a MATLAB struct), as this is only done once a new test suite is
-    % executed. To clear this persistent storage, run |clear functions|.
+    % executed. To clear this persistent storage, run `clear getReferenceHash`.
```

Comment:
Maybe we should decide on which markdown we chose: GitHub flavor (currently) or MATLAB flavor (before this change). I have no real preference, but we might be able to generate documentation using MATLAB flavored markdown in MATLAB itself.